syncthing.fc

```diff
@@ -0,0 +1,8 @@
+# binary
+/usr/bin/syncthing                      -- gen_context(system_u:object_r:syncthing_exec_t,s0)
+/usr/local/bin/syncthing                -- gen_context(system_u:object_r:syncthing_exec_t,s0)
+HOME_DIR/.local/bin/syncthing           -- gen_context(system_u:object_r:syncthing_exec_t,s0)
+
+# config files
+HOME_DIR/\.config/syncthing(/.*)?       -d gen_context(unconfined_u:object_r:syncthing_config_home_t,s0)
+HOME_DIR/\.config/syncthing(/.*)?       -- gen_context(unconfined_u:object_r:syncthing_config_home_t,s0)
```

Comment:
This can be a single entry, also use system_u instead of unconfined_u. `genhomedircon` has system_u hard-coded and so it would not be smart enough to deal with this even if this would make sense  (which it doesnt):
```
HOME_DIR/\.config/syncthing(/.*)?       gen_context(system_u:object_r:syncthing_config_home_t,s0)
```